HAL/pico/src/comms/XInputBackend.cpp

```diff
@@ -40,10 +40,12 @@ void XInputBackend::SendReport() {
     _report.b = _outputs.b;
     _report.x = _outputs.x;
     _report.y = _outputs.y;
-    _report.lb = _outputs.buttonL;
-    _report.rb = _outputs.buttonR;
-    _report.lt = _outputs.triggerLDigital ? 255 : _outputs.triggerLAnalog;
-    _report.rt = _outputs.triggerRDigital ? 255 : _outputs.triggerRAnalog;
+
+    _report.lb = _outputs.triggerLDigital;
+    _report.rb = _outputs.triggerRDigital;
+    _report.lt = _outputs.buttonL ? 255 : _outputs.triggerLAnalog;
+    _report.rt = _outputs.buttonR ? 255 : _outputs.triggerRAnalog;
```

Comment:
These seemed to not be working until flipped (lt <-> lb, rt <-> rb).
255 is the max value for when the digital press is registered.

If there are any concerns or issues, let me know. I was testing these settings from yuzu and some steam games.